.pre-commit-config.yaml

```
@@ -1,14 +1,13 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.3.0
-    hooks:
-      - id: black
-        args: ["--check", "--diff"]
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version.
-    rev: v0.0.275
+    rev: v0.1.7
     hooks:
+      # Run the linter.
       - id: ruff
+      # Run the formatter.
+      - id: ruff-format
+        args: ["--check"]
```

Is there a reason you don't use `--diff` as it was with `black`?
```suggestion
        args: ["--check", "--diff"]
```